CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Introduce flag `--show-inputs-only` to show the inputs of the command
```

Comment:
`Introduce` -> `Introduced` - please use past tense for consistency with other changelog messages 🙂

Reply:
fixe**D**

Reply:
side note - and I'm being a little difficult here - if this is a standard it should be documented in `contributing.md`.  if it is, it's fine to link it.  otherwise, these things feel arbitrary and unknowable to casual contributors.